### ISSUE TITLE
Add to the docs that ts-migrate-full will make git commits for you

### ISSUE THREAD

**Issue by isaacwashere** [airbnb/ts-migrate#199]
### General
**TLDR**:  It would be helpful to have a note somewhere in the documentation that when you run `ts-migrate-full`, the package will make a git commit for you at each major step in the process

### Explanation
I think an explicit line should be added somewhere in the documentation (likely around where it's expressed that running `ts-migrate-full` is recommended for entire projects) that when you run `ts-migrate-full`, a git commit will be authored upon completion of each major step.

I don't think this type of behavior from a package is common. At least, I don't recall installing packages, running them, and then having them make git commits for me. That is not to say that it isn't necessary in this case. I'm merely pointing out that since it isn't as common, it's unexpected. And that could lead to a slight surprise when running git commands after `...full` command is finished.

I'm curious on other's thoughts on this. Maybe this is behavior is obvious or I missed a section in the docs that point this out (and if that's the case, I apologize 😅 ). But I wanted to bring it up as a way to add a little more transparency into what's going on with the `ts-migrate-full` command.

**Comment by lencioni** [comment]
Hi Isaac! Thanks for the suggestion. This makes a lot of sense to me. Would you be willing to open a PR that updates the documentation to make this clearer?

**Comment by isaacwashere** [comment]
Yea, sure! I'd be happy to!
